integration_tests/features/support/steps.js

```diff
@@ -624,15 +544,22 @@ Then(
 
 Then(
   /node (.*) is at the same height as node (.*)/,
-  { timeout: 120 * 1000 },
+  { timeout: 130 * 1000 },
   async function (nodeA, nodeB) {
-    const expectedHeight = parseInt(await this.getClient(nodeB).getTipHeight());
-    await waitFor(
-      async () => this.getClient(nodeA).getTipHeight(),
-      expectedHeight,
-      115 * 1000
-    );
-    const currentHeight = await this.getClient(nodeA).getTipHeight();
+    var expectedHeight, currentHeight;
+    expectedHeight = parseInt(await this.getClient(nodeB).getTipHeight());
+    for (let i = 1; i <= 12; i++) {
```

Comment:
I don't understand how this helps? It definitely seems like the existing version should do exactly this? If it doesn't then the `waitFor(...)` implementation is wrong.

Reply:
`expectedHeight` from `nodeB` can change while the test is in progress, so that `expect(currentHeight).to.equal(expectedHeight);` never succeeds. This happens in some tests, especially with reorg tests, where the reference node is sometimes not on its final height when the test starts.